configs/virt/incus/scripts/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+IMAGE_NAME=skiffos/testing
+CONTAINER=skiff
+IMAGES_PATH="$BUILDROOT_DIR/images"
+
+command -v incus > /dev/null 2>&1 || {
+  echo "Failed to find the incus command. Is incus installed on your host system?"
+  exit 1
+}
+
+incus image show "${IMAGE_NAME}" > /dev/null || {
+  echo "Image ${IMAGE_NAME} not found. Did you execute \`make cmd/virt/incus/buildimage\`?"
+  exit 1
+}
+
+
+storage_pool="$(incus storage list -c n -f csv | head -n 1)"
+incus storage volume show "$storage_pool" skiff-persist >/dev/null 2>&1 || incus storage volume create "$storage_pool" "skiff-persist"
```

Comment:
If you use the full "if ! incus storage volume show ..." Format here it might make it clearer what is going on. Minor point